doc/dev/test_proxy_troubleshooting.md

```diff
@@ -152,6 +153,20 @@ As noted in the [Fetch environment variables][env_var_section] section of the [m
 reading expected variables from an accepted `**kwargs` parameter is recommended instead so that tests will run as
 expected in either case.
 
+## PermissionError during startup
```

Comment:
not sure if it would be good to specify mac? probably doesn't matter too much :P

Reply:
I was considering mentioning Mac here, but the error and advice also might not be OS specific. I think I'll leave it general for now, but might update if this presents differently elsewhere 🙂

Reply:
sgtm!